components/ai_chat/resources/page/components/premium_suggestion/index.tsx

```diff
@@ -77,11 +85,20 @@ function PremiumSuggestion(props: PremiumSuggestionProps) {
         </ul>
       </div>
       {!aiChatContext.isMobile && (
-        <div className={styles.priceList}>
-          <button className={styles.priceButton} tabIndex={-1}>
-            <span className={styles.priceButtonLabel}>Monthly</span>
-            <span className={styles.price}>{pricingInfo}</span>
-          </button>
+        <div className={styles.priceListWrapper}>
+          <div className={styles.priceList}>
+            <button className={styles.priceButton} tabIndex={-1}>
```

Comment:
just a comment, but it feels super weird that these are buttons but they aren't interactive

Reply:
I was honestly thinking the same 😂, didn't know if we had a plan to have some sort of select logic here in the future. So was just following the pattern of the current implementation https://github.com/brave/brave-core/commit/3c8017f068c48eff0f0b44841906b00edf81b898#diff-4e6dac01e90a3591eb0a4a8c3b5097a2ddf759b4fa2cb342fd6b093b3afa45b0R77 Happy to change these both to just a `<div>` if we want.

Reply:
Feel free to do as a followup if/when you have time :smile: